collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/FirstLaunchDialog.kt

```diff
@@ -66,7 +65,6 @@ class FirstLaunchDialog : MaterialFullScreenDialogFragment() {
         binding.configureLaterButton.setOnClickListener {
             projectImporter.importDemoProject()
             currentProjectProvider.setCurrentProject(ProjectImporter.DEMO_PROJECT_ID)
-            Collect.resetDatabaseConnections()
```

Comment:
You should be able to get rid of it now in normal code. It's still in tests so that we clean up connections during a full suite run rather than collecting thousands o them.